### DISCUSSION TITLE
Hassio Add-ons: Printing QR Codes in the Log tab

### DISCUSSION THREAD

**Discussion by w35l3y** [home-assistant/frontend#11491] [Other feature requests] [answered]
Any chance to make `line-height: normal` in the Log tab ?

So we could print QR Codes just like so...

| Expected | Actual |
| :---: | :---: |
| ![Expected QR Code without blank space between lines](https://user-images.githubusercontent.com/963510/151724263-82efab80-d941-483f-937d-479233f4fb1e.png) | ![Actual QR Code with blank spaces between lines](https://user-images.githubusercontent.com/963510/151724261-8e4d1918-e233-462a-a908-a65889840d71.png) |

**Comment by Misiu** [comment]
@w35l3y this is for your WhatsApp addon (https://github.com/w35l3y/hassio-addons/tree/main/whatsapp)?
I didn't know we could print QR codes in the console, this looks awesome 💪

**Reply by w35l3y** [reply]
Yes 😀

**Comment by matthiasdebaat** [comment, marked as answer]
HomeKit creates a [persistent notification](https://www.home-assistant.io/integrations/homekit/#setup) as integration. Could that also be a solution for add-ons? Changing the line height makes it harder to read.

**Reply by w35l3y** [reply]
I haven't thought about that. Thanks!

**Comment by tocaboca7864** [comment]
this look very good by the way I recently started using [OG WhatsApp](https://gbogfmyo.com/og-whatsapp/) from this site, and it’s honestly a game-changer. The extra features and privacy options make a huge difference. Highly recommend checking it out if you want more from your messaging app!
